charts/airflow/README.md

```diff
@@ -349,6 +349,33 @@ airflow:
 <hr>
 </details>
 
+<h3>Option 3 - Private PyPI Cloud Server</h3>
```

Comment:
I am not sure that this is truly distinct from Option-1, as it still uses the pip install init-containers, am I wrong?

Reply:
Right. I moved it to be Option 1a
I also gave an example of the pip-conf secret.
There're 3 steps that describe how to achieve it.